### PR TITLE
Support osram 2 button

### PR DESCRIPTION
**Pull request by dhylands**

### CODE REVIEW

File: zb-classifier.js [hunk 1]
```diff
@@ -403,6 +416,63 @@ class ZigbeeClassifier {
     );
   }
 
+  addButtonOnProperty(node, genOnOffOutputEndpoint) {
+    const property = this.addProperty(
+      node,                           // device
+      'on',                           // name
+      {                               // property description
+        '@type': 'BooleanProperty',
```
Comment (mrstegeman) [inline zb-classifier.js]: Couldn't this be `PushedProperty`?

File: zb-classifier.js [hunk 2]
```diff
@@ -827,6 +921,7 @@ class ZigbeeClassifier {
         '@type': 'LowPatteryProperty',
```
Comment (mrstegeman) [inline zb-classifier.js]: s/Pattery/Battery/

File: zb-classifier.js [hunk 3]
```diff
@@ -1088,6 +1195,14 @@ class ZigbeeClassifier {
     }
   }
 
+  initMultiLevelButton(node, genLevelCtrlOutputEndpoint) {
+    node.name = `${node.id}-button`;
+    node.type = 'multiLevelButton';
+    node['@type'] = ['MultiLevelButton'];
```
Comment (mrstegeman) [inline zb-classifier.js]: If you change the other property to `PushedProperty`, you could add `PushButton` here.

File: zb-node.js [hunk 4]
```diff
@@ -793,6 +863,146 @@ class ZigbeeNode extends Device {
...
+    const clusterId = parseInt(frame.clusterId, 16);
+    for (const property of this.properties.values()) {
+      if (profileId == property.profileId &&
+          endpoint == property.endpoint &&
+          clusterId == property.clusterId) {
+        switch (frame.zcl.cmdId) {
+          case 'on':   // on property
+            this.handleButtonOnOffCommand(property, true);
+            break;
+          case 'off': // on property]
```
Comment (mrstegeman) [inline zb-node.js]: nit: remove ]

File: zb-classifier.js (hunk 3 above)
Comment (dhylands) [reply zb-classifier.js]: But it isn't really a push button. There are 2 buttons. You push one of them to turn it on, and you push the other one to turn it off. You push and hold the on button to increase the level and push and hold the off button to decrease the level.

So it feels quite different from a push button to me.

File: zb-classifier.js (hunk 1 above)
Comment (dhylands) [reply zb-classifier.js]: It could, but it didn't feel right given the way that the switch is used (see below)

Comment (mrstegeman) [reply zb-classifier.js]: Ah, ok. I didn't know what the hardware actually was.

**Pull request merged**